Today/Controllers/ReminderDetailViewController/ReminderDetailEditDataSource.swift

```diff
@@ -78,6 +78,33 @@ class ReminderDetailEditDataSource: NSObject {
         }
         return cell
     }
+}
+
+// MARK: - Data Source Methods
+
+extension ReminderDetailEditDataSource: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        ReminderDetailSection.allCases.count
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        ReminderDetailSection(rawValue: section)?.numOfRowsInSection ?? 0
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        dequeueAndConfigureCell(for: indexPath, from: tableView)
+    }
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        guard let section = ReminderDetailSection(rawValue: section) else {
+            return nil
+        }
+        return section.sectionTitle
+    }
+    func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return false
+    }
```

Comment:
Don't need to override it as the default value is false, I guess

Reply:
I think the default is true, as when i removed this method it showed the "red-dash" marks...